app/src/main/java/io/multy/util/CryptoFormatUtils.java

```diff
@@ -27,7 +27,14 @@ public static String satoshiToBtc(long satoshi) {
         return result;
     }
 
-    public static String satoshiToUsd(long satoshi) {
+    public static double satoshiToBtcDouble(long satoshi) {
```

Comment:
Heard about method overloading? Also test first. These changes affect too much places.

Reply:
please, try to see full file https://github.com/Anschutz1927/Multy-android/blob/b8bae6c8ce9550efe12ec893340d50b4c3810acc/app/src/main/java/io/multy/util/CryptoFormatUtils.java